tests/ciscotest.rb

```diff
@@ -66,10 +67,15 @@ def interfaces
     unless @@interfaces
       # Build the platform_info, used for interface lookup
       # rubocop:disable Style/ClassVars
-      platform_info = PlatformInfo.new(node.host_name, :nexus)
-      @@interfaces = platform_info.get_value_from_key('interfaces')
+      @@interfaces = []
+      Interface.interfaces.each do |int, _obj|
```

Comment:
nit: equivalent is:

`      Interface.interfaces.keys.each do |int|`

Fine as is though.